File: mongodb-manual/mongo/100-print-01.py
#!/usr/bin/env python3
from pymongo import MongoClient
from pprint import pprint

client = MongoClient("mongodb://localhost")
db = client['mongodb-examples']

for d in db.restaurants.find({}).limit(5):
  print(d)

print("~"*30, "\n")

for d in db.restaurants.find({}).limit(5):
  pprint(d)

print("~"*30, "\n")

cursor = db.restaurants.find({}).limit(5)
for d in cursor:
  pprint(d)

File: mongodb-manual/mongo/100-print-01.js
db = db.getSiblingDB('mongodb-examples');

db.restaurants.find({}).limit(5)
db.restaurants.find({}).limit(5).pretty()

var cursor = db.restaurants.find({}).limit(5)
cursor.forEach(function(myDoc) { printjson(myDoc); } )

// :-/
var cursor = db.restaurants.find({}).limit(5)
cursor.toArray().slice(0,3).forEach(function(myDoc) { printjson(myDoc); } )
